modules/gitops-release-notes-1-5-0.adoc

```diff
@@ -0,0 +1,53 @@
++
+On a non-OpenShift cluster (meaning the Route API is not available), if you choose to enable the route, a log appears saying that the routes are not available in non-OpenShift environments and to please use ingresses instead. The state of the application controller and of the URL will not be affected.
++
+When the route or ingress is configured, but the corresponding controller is not set up properly (i.e. is not in Ready state or does not propagate its URL), this is indicated in the Operand as well in the value of `.status.url` as `Pending` instead of the URL. Also, if `.status.url` is `Pending`, this affects the overall status for the Operand by making it `Pending` instead of `Available`. link:https://issues.redhat.com/browse/GITOPS-654[GITOPS-654]
+
+
+[id="fixed-issues-1-5-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, RBAC rules specific to *AppProjects* would not allow the use of commas for the role's subject field, thus preventing bindings to LDAP account. This has been fixed and complex role bindings can now be specified in *AppProject* specific RBAC rules. link:https://issues.redhat.com/browse/GITOPS-1771[GITOPS-1771]
+
+* Before this update, when a `DeploymentConfig` is scaled to `0`, ArgoCD used to display it in *progressing* state with health status message as *"replication controller is waiting for pods to run"*. With this update, the edge case is fixed and the health check will report the correct health status of `DeploymentConfig` resource. link:https://issues.redhat.com/browse/GITOPS-1738[GITOPS-1738]
+
+* Before this update, the TLS certificate in the `argocd-tls-certs-cm` configuration map was deleted by the {gitops-title} unless the certificate was configured in the ArgoCD CR specification `tls.initialCerts` field. This update fixes this issue. link:https://issues.redhat.com/browse/GITOPS-1725[GITOPS-1725]
+
+* Before this update, while creating a namespace with the `manage-by` label it created a lot of `RoleBinding` on the new namespace. This update fixes the issue of {gitops-title} removes the extraneous `Role` and `RoleBinding` resources that were created by the previous versions.
+ link:https://issues.redhat.com/browse/GITOPS-1550[GITOPS-1550]
+
+* Before the update,  Firefox 94 and later failed to connect to Argo CD UI with error code *SEC_ERROR_BAD_DER* since TLS certificate of the Route in pass-through mode did not not had a CA name.
+
+With this update, first delete the `<argocd-instance-name>-tls` secrets once the {gitops-title} is recreated. After recreating again by the {gitops-itle}, the Argo CD UI is accessible by Firefox. link:https://issues.redhat.com/browse/GITOPS-1548[GITOPS-1548]
```

Comment:
First, delete openshift-gitops-ca, let it recreate, then delete openshift-gitops-tls, then it works. 

@DebarghoGhosh I have commented on the JIRA to correct the release notes accordingly.